experiments/memcached-cat/main.go

```diff
@@ -48,6 +48,7 @@ var (
 	qpsFlag                  = conf.NewStringFlag("cat_qps", "Comma-separated list of QpS to iterate over", "375000")
 	maxCacheWaysToAssignFlag = conf.NewIntFlag("cat_max_cache_ways", "Mask representing maximum number of cache ways to assign to a job. It is assumed that cat_max_cache_ways and cat_min_cache_ways sum to number of all cache ways available.", 11)
 	minCacheWaysToAssignFlag = conf.NewIntFlag("cat_min_cache_ways", "Mask representing minumim number of cache ways to assing to a job. It is assumed that cat_max_cache_ways and cat_min_cache_ways sum to number of all cache ways available.", 1)
+	cacheParitioningFlag     = conf.NewBoolFlag("cat_cache_paritioning", "Enable separate cache parition for HP workload (otherwise HP has access to whole cache, not just rest of not used by BE workload)", false)
```

Comment:
s/Enable separate cache parition for HP workload/Enables dedicated sets of cache ways for HP and BE workloads
s/(otherwise HP has access to whole cache, not just rest of not used by BE workload)/(if disabled then HP workload uses all cache ways all the time)